pkgs/development/libraries/flutter-engine/default.nix

```diff
@@ -0,0 +1,283 @@
+          };
+
+          dontUnpack = true;
+          dontConfigure = true;
+          dontBuild = true;
+
+          ## FIXME: use "flutter-deps" and the arguments which the OG script takes to generate the git revision
+          gitrev = writeText "git-revision.py" ''
+            #!${python3}/bin/python3
+            print("${fakeHash}")
+          '';
+
+          dartPackageConfig = writeText "dart-package-config.json" ''
+            {
+              "configVersion": 2,
+              "generator": "tools/generate_package_config.dart",
+              "packages": []
+            }
+          '';
+
+          binaryFixes = [
+            "src/third_party/dart/tools/sdks/dart-sdk/bin/dart"
+            "src/flutter/third_party/gn/gn"
+            "src/buildtools/linux-x64/clang/bin/clang++"
+            "src/buildtools/linux-x64/clang/bin/clang-apply-replacements"
+            "src/buildtools/linux-x64/clang/bin/clang-doc"
+            "src/buildtools/linux-x64/clang/bin/clang-format"
+            "src/buildtools/linux-x64/clang/bin/clang-include-fixer"
+            "src/buildtools/linux-x64/clang/bin/clang-refactor"
+            "src/buildtools/linux-x64/clang/bin/clang-scan-deps"
+            "src/buildtools/linux-x64/clang/bin/clang-tidy"
+            "src/buildtools/linux-x64/clang/bin/clangd"
+            "src/buildtools/linux-x64/clang/bin/dsymutil"
+            "src/buildtools/linux-x64/clang/bin/find-all-symbols"
+            "src/buildtools/linux-x64/clang/bin/lld"
+            "src/buildtools/linux-x64/clang/bin/llvm-ar"
+            "src/buildtools/linux-x64/clang/bin/llvm-bolt"
+            "src/buildtools/linux-x64/clang/bin/llvm-cov"
+            "src/buildtools/linux-x64/clang/bin/llvm-cxxfilt"
+            "src/buildtools/linux-x64/clang/bin/llvm-debuginfod-find"
+            "src/buildtools/linux-x64/clang/bin/llvm-dwarfdump"
+            "src/buildtools/linux-x64/clang/bin/llvm-dwp"
+            "src/buildtools/linux-x64/clang/bin/llvm-gsymutil"
+            "src/buildtools/linux-x64/clang/bin/llvm-ifs"
+            "src/buildtools/linux-x64/clang/bin/llvm-libtool-darwin"
+            "src/buildtools/linux-x64/clang/bin/llvm-lipo"
+            "src/buildtools/linux-x64/clang/bin/llvm-mt"
+            "src/buildtools/linux-x64/clang/bin/llvm-nm"
+            "src/buildtools/linux-x64/clang/bin/llvm-objcopy"
+            "src/buildtools/linux-x64/clang/bin/llvm-objdump"
+            "src/buildtools/linux-x64/clang/bin/llvm-pdbutil"
+            "src/buildtools/linux-x64/clang/bin/llvm-profdata"
+            "src/buildtools/linux-x64/clang/bin/llvm-rc"
+            "src/buildtools/linux-x64/clang/bin/llvm-readobj"
+            "src/buildtools/linux-x64/clang/bin/llvm-size"
+            "src/buildtools/linux-x64/clang/bin/llvm-symbolizer"
+            "src/buildtools/linux-x64/clang/bin/llvm-undname"
+            "src/buildtools/linux-x64/clang/bin/llvm-xray"
```

Comment:
refactor

```sh
for f in $out/src/buildtools/linux-x64/clang/bin/*; do
  ...
done
```

Reply:
Fixed so now it ignores symlinks but fixes binaries and scripts since that directory contains both.